### ISSUE TITLE
fix(overflow-menu): keyboard support for Overflow Menu

### ISSUE THREAD

**Issue by lovemecomputer** [carbon-design-system/carbon#1119]
### Summary

Implement keyboard up&down arrow functionality, to compliment the a11y improvements in https://github.com/IBM/carbon-components/pull/1115

### Justification

a11y

### Desired UX and success metrics

The overflow menu should be behave in the expected way for both screen-reader users, and general UI keyboard input expectations.
